python/ql/src/experimental/typetracking/TypeTracker.qll

```diff
@@ -0,0 +1,420 @@
+/** Step Summaries and Type Tracking */
+
+private import TypeTrackerPrivate
```

Comment:
For other uses of pyrameterized modules, we have typically called this `TypeTrackerSpecific.qll`.

Reply:
Done.